### DISCUSSION TITLE
Naming Convention request for DateTime at end of BaseName

### DISCUSSION THREAD

**Discussion by GShrub** [EricZimmerman/KapeFiles#556] [Ideas]
This is a minor request.
I would like to be able to choose the naming convension of the vhdx/zip file currently the file names are DateTime_BaseName. It would be convenient if there was an option for BaseName_DateTime as well. 
If I have missed in the documentation on how to achieve please accept my appologies and slap me with a RTFM and point me in the right direction.
Love your work guys.

**Comment by EricZimmerman** [comment]
there is not an option for that currently. the timestamp is used to make sorting easier, etc and know when things were collected. why are you wanting to have the basename first? grouping?

the other thing is, i would have to do the same for the console log, etc since it too uses the timestamp for grouping, so this is not a trivial thing.

**Reply by GShrub** [reply]
Hi Eric,
Thanks for responding, you are correct it is for grouping multiple images of systems togeather and having multaple systems alphabetically sorted in a folder is convieniant. It is by no means a show stopper. 

I appreciate that minor request doesn't mean there is a simple solution which is why it can't be implemented. I appreciate you coming back to me and also the explanation.

All the best
